include/View.hpp

```diff
@@ -30,10 +30,14 @@ struct View : public ViewListeners
 
   void close();
 
+  struct wlr_output *getOutput();
+
   Server *server;
   struct wlr_xdg_surface_v6 *xdg_surface;
   bool mapped;
   int x, y;
+  bool fullscreen;
+  wlr_box saved;
```

Comment:
Would maybe make more sense to store this in the `Output` class ?
That way we don't store it once per view, and since there can only be one fullscreen window per output we shouldn't need more than that.

Reply:
(this may require finding the right Output containing the wlr_output* in the Output vector in ServerOutput)